lib/fastlane/actions/slather.rb

```diff
@@ -135,7 +139,8 @@ def self.available_options
                                        optional: true),
           FastlaneCore::ConfigItem.new(key: :ignore,
                                        env_name: "FL_SLATHER_IGNORE",
-                                       description: "Tell slather to ignore files matching a path",
+                                       description: "Tell slather to ignore files matching a path or any path from an array of paths",
+                                       is_string: false,
```

Comment:
I believe this is kind of type safety in `ConfigItem` module (or class?), but it only checks for `String` type.
The default is `true`, so if I take it out and pass an array to `slather` action, the error will be this

```
[08:15:51]: You passed invalid parameters to 'slather'.
[08:15:51]: Check out the error below and available options by running `fastlane action slather`
[08:15:51]: error: coverage, 'ignore' value must be a String! Found Array instead.
```

The problem is that we have a param which can be of multiple types. And even when `is_string` is `false`, we still can pass a string and it works...

Anyway, I could add a warning that will show up in fastlane log when people use `String` type. How does that sound? 

Alternatively, I could add one more parameter that takes an array of paths and add deprecation message for the old one.